bftclient/include/bftclient/config.h

```diff
@@ -0,0 +1,89 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License"). You may not use this product except in
+// compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright notices and license terms. Your use of
+// these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include <chrono>
+#include <map>
+#include <set>
+#include <string>
+#include <variant>
+#include <vector>
+
+#include "bftclient/base_types.h"
+#include "bftclient/quorums.h"
+
+using namespace std::chrono_literals;
```

Comment:
Please remove this from the include file

Reply:
Doing that means making all the defaults much more verbose. I'd be really surprised if this caused serious problems for callers. I think the readability of the existing usage outweighs any risk of some user wanting to define their own "ms" or "s" suffixes.